src/pages/Profile.tsx

```diff
@@ -1,5 +1,219 @@
+import React from "react";
+import { useEffect } from "react";
+import KeyboardBackspaceIcon from "@mui/icons-material/KeyboardBackspace";
+import { Button, Typography, Tabs, Tab, Avatar, Box } from "@mui/material";
+import EditIcon from "@mui/icons-material/Edit";
+import ProfileTweets from "../components/Posts/ProfileTweets";
+import axios from "axios";
+import { useAppSelector } from "../state/hooks";
+import ProfileReplies from "../components/Posts/ProfileReplies";
+import ProfileLikes from "../components/Posts/ProfileLikes";
+import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
+import { useNavigate } from "react-router-dom";
+import IconButton from "@mui/material/IconButton/IconButton";
+
+const styles = {
+  arrowBtn: {
+    border: "none",
+    backgroundColor: "white",
+  },
+  arrowText: {
+    fontSize: 20,
+    fontWeight: "bold",
+  },
+  avatar: {
+    width: 140,
+    height: 140,
+    left: 25,
+    zIndex: 2,
+    top: -80,
+  },
+  editBtn: {
+    backgroundColor: "black",
+    borderRadius: 20,
+    textTransform: "none",
+    marginLeft: 62,
+    marginTop: 2,
+  },
+  displayName: {
+    fontSize: 25,
+    marginTop: 10,
+    marginBottom: 5,
+    fontWeight: "bold",
+  },
+  usernameDisplay: {
+    fontWeight: "normal",
+    fontSize: 20,
+    color: "#71797E",
+    marginBottom: 15,
+  },
+  menu: { marginTop: 60, marginRight: 83 },
+  tabSelection: {
+    marginLeft: 85,
+  },
+  header: {
+    height: "60px",
+    paddingTop: 10,
+    display: "flex",
+    alignItems: "center",
+  },
+  tweetCountStyle: { fontWeight: "normal", fontSize: 15 },
+  followercountbtn: {
+    border: "none",
+    fontSize: 15,
+    padding: 0,
+    backgroundColor: "white",
+    color: "black",
+    textTransform: "none",
+    marginTop: 2,
+  },
+  followingcountbtn: {
+    border: "none",
+    fontSize: 15,
+    padding: 0,
+    marginLeft: 5,
+    color: "black",
+    backgroundColor: "white",
+    textTransform: "none",
+    marginTop: 2,
+  },
+  bioDisplay: {
+    marginBottom: 20,
+  },
+  parentBox: {
+    width: 650,
+  },
+  avatarBannerBox: {
+    height: 200,
+  },
+  profileInfoBox: {
+    paddingLeft: 15,
+    paddingTop: 20,
+  },
+  calendarBox: {
+    alignItems: "center",
+    display: "flex",
+  },
+};
+
 const Profile = () => {
-  return <h1 style={{ margin: "auto" }}>Profile</h1>;
+  const navigate = useNavigate();
+  const [value, setValue] = React.useState("one");
+  const [profileContents, setProfileContents] = React.useState({
+    tweetcount: "",
```

Comment:
Can we please rename these to be camel case? Same goes for the backend the returns them